src/components/navbar.component.tsx

```diff
@@ -43,44 +54,74 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout }) => {
     }
   }, [displayChangeLocationUI, toggleChangeLocationUI]);
 
-  React.useEffect(() => {
-    if (sidenavOpen) {
-      document.body.classList.add("omrs-sidenav-expanded");
-      return () => document.body.classList.remove("omrs-sidenav-expanded");
-    }
-  }, [sidenavOpen]);
-
   const reLoadLocation = (): void => {
     setDisplayChangeLocationUI(false);
   };
 
   return (
     <>
-      <nav className={styles.topNav}>
-        <Link to={`${openmrsSpaBase}home`}>
-          <svg className="omrs-icon">
-            <use xlinkHref="#omrs-icon-home" />
-          </svg>
-        </Link>
-        <div className={`omrs-type-title-4 ${styles.logo}`}>
-          <svg role="img" width="10rem">
-            <use xlinkHref="#omrs-logo-partial-mono" />
-          </svg>
-        </div>
-        <Location
-          reLoadLocation={displayChangeLocationUI}
-          toggleChangeLocationUI={toggleChangeLocationUI}
-        />
-        <Details user={user} onToggle={toggleMenu} />
-      </nav>
-      <UserMenu user={user} open={isUserMenuOpen}>
-        <Logout onLogout={onLogout} />
-      </UserMenu>
-      {displayChangeLocationUI && (
-        <ChangeLocation refreshLocation={reLoadLocation} />
-      )}
+      <HeaderContainer
+        render={({ isSideNavExpanded, onClickSideNavExpand }) => (
+          <Header aria-label="OpenMRS">
+            <HeaderMenuButton
+              aria-label="Open menu"
+              isCollapsible
+              onClick={onClickSideNavExpand}
+              isActive={isSideNavExpanded}
+            />
+            <HeaderName prefix="">OpenMRS</HeaderName>
+
+            <HeaderGlobalBar>
+              <HeaderGlobalAction
+                aria-label="Search"
+                aria-labelledby="Search Icon"
```

Comment:
Right now nothing happens here, right?

Should we keep the icon or drop it (if nothing happens on clicking it there is not much sense to it).

Reply:
+1 to drop